src/_includes/head.html

```diff
@@ -61,14 +61,19 @@
   {% if isDocumentation %}
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/css/perfect-scrollbar.min.css">
     <script defer type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/js/perfect-scrollbar.min.js"></script>
+    <script defer type="text/javascript" src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.3.min.js"></script>
+    <script defer src="{{ "/scripts/widget.js" | prepend: site.baseurl }}" type="text/javascript"></script>
```

Comment:
These lines were already there - don't know why they were included in this commit